packages/form-core/src/FieldApi.ts

```diff
@@ -1643,6 +1643,30 @@ export class FieldApi<
     )
   }
 
+  /**
+   * Parses the field's value with the given schema and returns
+   * issues (if any). This method does NOT set any internal errors.
+   * @param schema The standard schema to parse this field's value with.
+   */
+  parseValueWithSchema(schema: StandardSchemaV1) {
```

Comment:
Let's use the arrow function notation to preserve the scope (and consistency) in the new methods